code/modules/events/hostile_infestation.dm

```diff
@@ -0,0 +1,125 @@
+//Hostile Infestation: A variant of the infestation event that spawns small numbers of hostile mobs.
+//Currently only fires if there are 2 or more security officers
+
+#define LOC_KITCHEN 0
+#define LOC_ATMOS 1
+#define LOC_INCIN 2
+#define LOC_CHAPEL 3
+#define LOC_LIBRARY 4
+#define LOC_HYDRO 5
+#define LOC_VAULT 6
+#define LOC_TECH 7
+
+#define MONSTER_BEAR    0
+#define MONSTER_CREATURE 1
+#define MONSTER_XENO 2
+#define MONSTER_HIVEBOT  3
+#define MONSTER_ZOMBIE    4
+#define MONSTER_SKRITE  5
+#define MONSTER_SQUEEN  6
+
+/datum/event/hostile_infestation
+	announceWhen = 2  //Should notify crew much quicker than normal infestation event because of the danger posed by these mobs
+	endWhen = 10
+	var/localestring
+	var/monsterstring
+
+/datum/event/hostile_infestation/start()
+	to_chat(world, sound('sound/effects/bumpinthenight.ogg'))
```

Comment:
M.playsound_local(epicenter, 'sound/effects/explosionsmallfar.ogg', far_volume, 1, frequency, falloff = 5)

I'm not a very experienced coder and there's a lot of other stuff attached to this.
I used to_chat(world) because that's how it's done for the AI alerts for other events. Can I change "world" to something else to just get it to play on the z-level?